src/Phoenix.vue

```diff
@@ -57,4 +57,18 @@ export default {
     height: 100vh;
     overflow: hidden;
   }
+
+  .oc-page-title {
+    margin: 0;
+  }
+
+  .visually-hidden {
```

Comment:
There is already in place actually visually-hidden mixin - https://github.com/owncloud/owncloud-design-system/blob/0db4c32de7fa12a03de7b6aa0d20f09f11aa9ebd/src/styles/_mixins.scss#L56

Reply:
I see. Will create a utility class for this, using this mixin in the DS, then using mentioned class in phoenix if that's okay.

Reply:
The corresponding DS PR: https://github.com/owncloud/owncloud-design-system/pull/498
It will establish `.oc-visually-hidden` which I'll use here.